tests/manager/manager-maxstalenessms-001.phpt

```diff
@@ -1,13 +0,0 @@
---TEST--
-MongoDB\Driver\Manager: maxStalenessMS
```

Comment:
This was essentially renamed to `manager-ctor-read_preference-002.phpt`, where I added an additional test case and assert the state of `Manager::getReadPreference()`.